Mathlib/Algebra/Group/Nat/Even.lean

```diff
@@ -94,6 +94,10 @@ theorem add_one_lt_of_even {n m : ℕ} (hn : Even n) (hm : Even m) (hnm : n < m)
   rcases hm with ⟨m, rfl⟩
   omega
 
+theorem even_iff_iff_mod_two_eq : (Even n ↔ Even m) ↔ n % 2 = m % 2 := by
```

Comment:
I was confused by the name. I understood after reading the statement, but I didn't predict it from the name. Please update the PR title also.
```suggestion
theorem even_iff_even_iff_mod_two : (Even n ↔ Even m) ↔ n % 2 = m % 2 := by
```